tools/finish_release.py

```diff
@@ -214,6 +223,7 @@ def main(args):
 
     github_access_token_file = parsed_args.githubpat
     github_access_token = open(github_access_token_file, 'r').read().rstrip()
+    css = parse_args.css
 
     with tempfile.TemporaryDirectory() as tempdir:
         version = download_azure_artifacts(tempdir)
```

Comment:
You may have already picked up on this, but this function automates the process of finding and downloading the unsigned Windows installer for the release. This unsigned installer is not uploaded to GitHub until the `create_github_release` function below AFTER the `scp` command that this PR adds.

If this creates a problem for your imagined workflow, I wonder if we couldn't do the following in this script:

1. `scp` the unsigned installer downloaded here to the CSS.
2. `ssh` into the code signing server and run a script to sign the installer.
3. `scp` the signed installer locally and upload it.